de.fu_berlin.inf.dpp/src/de/fu_berlin/inf/dpp/editor/EditorPool.java

```diff
@@ -301,8 +304,11 @@ public void remove(final IEditorPart editorPart) {
 
     /**
      * Removes all {@link IEditorPart} from the EditorPool.
+     * 
+     * @param referencePointManager
+     *            TODO
      */
-    public void removeAllEditors() {
+    public void removeAllEditors(IReferencePointManager referencePointManager) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused method parameter "referencePointManager". ([squid:S1172](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3AS1172))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=276&issues=AWajNvBJrChX7_AlOxX4&open=AWajNvBJrChX7_AlOxX4)

[ISSUE_KEY]: AWajNvBJrChX7_AlOxX4